content/en/contributing/development-environment-setup.md

```diff
@@ -30,46 +30,80 @@ You will need the following tools for the local development of LocalStack.
 
 We recommend you to individually install the above tools using your favorite package manager. For example, on macOS, you can use [Homebrew](https://brew.sh/) to install the above tools.
 
-### Setting up Development Environment
+### Setting up the Development Environment
 
-To make contributions to LocalStack, you need to be able to run LocalStack in host mode from your IDE, and be able to attach a debugger to the running LocalStack instance. We have a basic tutorial to cover how you can do that.
+To make contributions to LocalStack, you need to be able to run LocalStack in host mode from your IDE, and be able to attach a debugger to the running LocalStack instance.
+We have a basic tutorial to cover how you can do that.
 
 The basic steps include:
 
-*   Clone the localstack repository, and optionally our fork of `moto`:
-    -   [https://github.com/localstack/localstack/](https://github.com/localstack/localstack/)
-    -   [https://github.com/localstack/moto](https://github.com/localstack/moto)
-*   Make sure you have `javac`, `node`, `npm`, and Python 3.10 installed.
-*   Most of the contributors use the free community version [PyCharm](https://www.jetbrains.com/pycharm/).
+1. Fork the localstack repository on GitHub [https://github.com/localstack/localstack/](https://github.com/localstack/localstack/)
+2. Clone the forked localstack repository `git clone git@github.com:<GITHUB_USERNAME>/localstack.git`
+3. Ensure you have `python` and `pip` installed.
+   > You might also need `node`, `npm`, and `javac` for some emulated services.
+4. Install the Python dependencies using `make install`.
+   > This will install the required pip dependencies in a local Python `virtualenv` directory `.venv` (your global python packages will remain untouched).
+   > Depending on your system, some `pip`/`npm` modules may require additional native libs installed. 
+5. Start localstack in host mode using `make start`
 
 {{< youtube XHLBy6VKuCM >}}
 
-If you pull the repo in order to extend/modify LocalStack, run this command to install all the dependencies:
-
-{{< command >}}
-$ make install
-{{< /command >}}
+### Building the Docker image for Development
 
-This will install the required pip dependencies in a local Python `virtualenv` directory `.venv` (your global python packages will remain untouched), as well as some node modules in `./localstack/node_modules/`. Depending on your system, some `pip`/`npm` modules may require additional native libs installed.
+Please note that there are a few commands we need to run on the host to prepare the local environment for the Docker build - specifically,
```

Comment:
@alexrashed Isn't this outdated advice with LPM?

Reply:
Good catch! Yes, this is definitely outdated (for quite a while now 😅)!

Reply:
there was even an outdated comment at the top 🙈